frontend/src/modules/resourceprogress/components/ProgressChart.tsx

```diff
@@ -92,7 +92,7 @@ export default class ProgressChart extends React.Component<Props> {
         const x = canvas.width / 2;
         const y = canvas.height / 2;
         const radius = (canvas.width - context.lineWidth) / 2;
-        let end = null;
+        let end: number = null;
```

Comment:
I was wondering if I could set `end = -0.5` and be done, but I had no idea how to actually check if my reading of the code was right.

Reply:
Yup, I think you are right. Please also update the logic in `progress-chart.js`.